### PR TITLE
Add cask "openin" v3.0.3

### PR DESCRIPTION
**Pull request by singularitti**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/openin.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+cask "openin" do
+  version "3.0.3"
+  sha256 "e6f17699c9cb59f53c17b07f00b225dcfdd15fefc2e967ebd4999558963f9529"
+
+  url "https://loshadki.app/openin/OpenIn%20#{version}.dmg"
+  name "openin"
```
Comment (neutric) [inline Casks/openin.rb:6]:
```
name "OpenIn"
```

File: Casks/openin.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  desc "Open links/files with specific browsers/apps"
+  homepage "https://loshadki.app/openin/"
+
+  livecheck do
+    url :homepage
+    strategy :page_match
```
Comment (neutric) [inline Casks/openin.rb:12]: `strategy :page_match` should only be used when explicitly needed. It is the default except for special cases such as GitHub-URLs.

File: Casks/openin.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    regex(%r{href=.*?/OpenIn%20(\d+(?:\.\d+)*)\.dmg}i)
```
Comment (neutric) [inline Casks/openin.rb:13]:
```
regex(/href=.*?OpenIn%20v?(\d+(?:\.\d+)+)\.dmg/i)
```

File: Casks/openin.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  end
+
```
Comment (neutric) [inline Casks/openin.rb:15]:
```
depends_on macos: ">= :big_sur"
```

**Review by neutric — COMMENTED** [review]
Thanks for your contribution to `homebrew-cask`, @singularitti! My review is in the code. Take it with a grain of salt as I am not a maintainer.

**Comment by singularitti** [comment]
Thank you! @neutric

**Pull request merged**